src/app/pages/help/index.js

```diff
@@ -1,59 +1,61 @@
-import { Page } from '../../components/page';
-import { SectionContainer, SectionContent, SectionHeader } from '../../components/section';
-import help from '../../data/help';
-import { Button, Card, Title } from "@newfold/ui-component-library";
+import { Button, Card, Title } from '@newfold/ui-component-library';
+import { Page } from 'App/components/page';
+import {
+	SectionContainer,
+	SectionContent,
+	SectionHeader,
+} from 'App/components/section';
+import help from 'App/data/help';
 
-const HelpCard = ({ item }) => {
-    return ( 
-        <Card className={`wppbh-help-card card-help-${item.name}`}>
-            <Card.Content>
-                <Title 
-					as="h3" 
-					size="4"
-					className="nfd-mb-2"	
-				>{item.title}</Title>
-                <p>{item.description}</p>
-            </Card.Content>
+const HelpCard = ( { item } ) => {
+	return (
+		<Card className={ `wppbh-help-card card-help-${ item.name }` }>
+			<Card.Content>
+				<Title as="h3" size="4" className="nfd-mb-2">
+					{ item.title }
+				</Title>
+				<p>{ item.description }</p>
+			</Card.Content>
 
-            <Card.Footer>
-                <Button
-                    variant="secondary"
-                    as="a"
-                    className="nfd-w-full"
-                    href={item.url}
-                    target="_blank"
-                >
-                    {item.cta}
-                </Button>
-            </Card.Footer>
-        </Card>
-     );
-}
+			<Card.Footer>
+				<Button
+					variant="secondary"
+					as="a"
+					className="nfd-w-full"
+					href={ item.url }
+					target="_blank"
+				>
+					{ item.cta }
+				</Button>
+			</Card.Footer>
+		</Card>
+	);
+};
 
 const Help = () => {
 	const renderHelpCards = () => {
 		const helpItems = help;
 
 		return (
 			<div className="nfd-grid nfd-gap-6 nfd-grid-cols-1 sm:nfd-grid-cols-2 xl:nfd-grid-cols-3 2xl:nfd-grid-cols-4">
-				{helpItems.map((item) => (
-					<HelpCard key={item.name} item={item} />
-				))}
+				{ helpItems.map( ( item ) => (
+					<HelpCard key={ item.name } item={ item } />
+				) ) }
 			</div>
 		);
 	};
 	return (
-		<Page className={"wppbh-app-help-page"}>
-			<SectionContainer className={'wppbh-app-help-container'}>
+		<Page className={ 'wppbh-app-help-page' }>
+			<SectionContainer className={ 'wppbh-app-help-container' }>
 				<SectionHeader
-					title={__('Help', 'wp-plugin-bluehost')}
-					subTitle={__('We are available 24/7 to help answer questions and solve your problems.',
-					'wp-plugin-bluehost')}
+					title={ __( 'Help', 'wp-plugin-bluehost' ) }
+					subTitle={ __(
+						'We are available 24/7 to help answer questions and solve your problems.',
```

Comment:
We use "24 hours - 7 days" elsewhere, should these be consistent?

_If this text comes from marketing or somewhere else, feel free to ignore._